### PR TITLE
FINAL Edits to mcstas-suite for build and execute on Windows via conda

### PR DESCRIPTION
**Pull request by willend**
…a-forge/mcstas-suite-feedstock/pull/12 / https://github.com/conda-forge/mcstas-suite-feedstock/pull/13

- And most of the earlier review comments taken into account

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR allows building / testing on the Windows platform - now from my private fork to allow re-rendering...
(It is yet another "clean slate" replacement of https://github.com/conda-forge/mcstas-suite-feedstock/pull/12 and https://github.com/conda-forge/mcstas-suite-feedstock/pull/13)

### CODE REVIEW

**Comment by willend** [comment]
@conda-forge-admin, please rerender

**Comment by willend** [comment]
@conda-forge-admin, please rerender

**Comment by willend** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
+{% set sha_value = "b13e5c0ee5ff2945920c6b401d57121962cec92dcb6ac9bc70d6d499d8f6c256" %}
 #NB get sha256 with: curl -sL <URL> | openssl sha256
 
 package:
   name: mcstas-suite
   version: {{ version }}
 
 source:
-  url: https://github.com/McStasMcXtrace/McCode/archive/mcstas-{{ version }}.tar.gz
+  url: https://github.com/willend/McCode/archive/mcstas-{{ version }}.tar.gz
```
Comment (tkittel) [inline recipe/meta.yaml:10]: Det her er vist stadigvæk den gale url, men det er måske med vilje?

**Comment by tkittel** [comment]
Diff looks good to me, except the url of course.

File: recipe/meta.yaml (hunk 1 above)
Comment (willend) [reply recipe/meta.yaml:10]: Jeps, det er med vilje. Jeg vil meget gerne lige se et byg der virker baseret på dette PR inden jeg laver et nyt tag på McStasMcXtrace/McCode (main)

**Comment by willend** [comment]
@tkittel suggested gameplan is:
1. We see the tests run through correctly
2. I create a new mcstas-3.4.9 tag on the main branch of the McCode repo
3. I adapt this PR for the new URL/hash
4. We merge

**Comment by willend** [comment]
Steps 1-3 done

**Comment by willend** [comment]
@tkittel "all systems go" it seems - merge?

**Pull request merged**